test/Middleware.js

```diff
@@ -31,8 +31,9 @@ describe('Middleware', () => {
       `.user h1:last-child{-webkit-clip-path:none;clip-path:none;}`,
     ].join(''))
 
-    expect(serialize(compile(`@keyframes name{from{width:0;}to{width:1;}}`), middleware([prefixer, stringify]))).to.equal([
-      `@-webkit-keyframes name{from{width:0;}to{width:1;}}`, `@keyframes name{from{width:0;}to{width:1;}}`
+    expect(serialize(compile(`@keyframes name{from{transform: rotate(0deg);}to{transform: rotate(360deg);}}`), middleware([prefixer, stringify]))).to.equal([
```

Comment:
i've changed this to showcase that contents of `@keyframes` and `@-webkit-keyframes` does not get duplicated by prefixer anyhow